Packs/PingIdentity/pack_metadata.json

```diff
@@ -0,0 +1,28 @@
+{
+    "name": "PingIdentity",
+    "description": "Integration with PingIdentity's PingOne identity platform",
+    "support": "partner",
+    "currentVersion": "1.0.0",
+    "author": "Peter Holko",
```

Comment:
Please change the author name to the vendor name.

Reply:
Updated to tap-global@pingidentity.com and the author to Ping Identity